Dockerfile.ios

```diff
@@ -10,31 +10,46 @@ RUN dnf -y install --setopt=install_weak_deps=False \
     git clone --progress https://github.com/tpoechtrager/cctools-port.git && \
     cd /root/cctools-port && \
     git checkout 634a084377ee2e2932c66459b0396edf76da2e9f && \
+    #iOS arm64 device
     usage_examples/ios_toolchain/build.sh /root/files/iPhoneOS${IOS_SDK}.sdk.tar.xz arm64 && \
     mkdir -p /root/ioscross/arm64 && \
     mv usage_examples/ios_toolchain/target/* /root/ioscross/arm64 && \
     mkdir /root/ioscross/arm64/usr && \
     ln -s /root/ioscross/arm64/bin /root/ioscross/arm64/usr/bin && \
+    #iOS arm64 simulator
+    sed -i 's/miphoneos-version-min/mios-simulator-version-min/g' usage_examples/wrapper.c && \
+    usage_examples/ios_toolchain/build.sh /root/files/iPhoneSimulator${IOS_SDK}.sdk.tar.xz arm64 && \
+    mkdir -p /root/ioscross/arm64_sim && \
+    mv usage_examples/ios_toolchain/target/* /root/ioscross/arm64_sim && \
+    mkdir /root/ioscross/arm64_sim/usr && \
+    ln -s /root/ioscross/arm64_sim/bin /root/ioscross/arm64_sim/usr/bin && \
```

Comment:
Seems like `osxcross` toolchain do not support ARM64 iOS Simulator target. Toolchain built with Simulator SDK produce incompatible iOS binaries, and it's not building with `arm64-apple-ios-simulator` target at all.

Reply:
Faced the same problem with WIP tvOS arm64 simulator build container.

There is an existing issue: https://github.com/tpoechtrager/cctools-port/issues/102 so it seems like arm64 support might appear in future.